### PR TITLE
sp cli: make sectors list much faster

### PR DESCRIPTION
**Pull request by magik6k**
## Proposed Changes
Make sector checks run in parallel

## Additional Info
```
pre
./lotus-miner sectors list  31.82s user 1.64s system 59% cpu **56.525 total**
./lotus-miner sectors list --fast  34.44s user 1.97s system 329% cpu **11.060 total**


post
./lsm sectors list  15.59s user 1.72s system 248% cpu **6.971 total**
./lsm sectors list --fast  11.24s user 2.76s system 784% cpu **1.785 total**
```

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: cmd/lotus-miner/sectors.go [hunk 1]
```diff
@@ -306,9 +310,14 @@ var sectorsListCmd = &cli.Command{
...
 		},
+		&cli.Int64Flag{
+			Name:  "check-parallelism",
+			Usage: "number of parallel requests to make for checking sector states",
+			Value: parallelSectorChecks,
+		},
 	},
 	Action: func(cctx *cli.Context) error {
-		minerApi, closer, err := lcli.GetStorageMinerAPI(cctx)
+		minerApi, closer, err := lcli.GetStorageMinerAPI(cctx, cliutil.StorageMinerUseHttp)
```
Comment (arajasek) [inline cmd/lotus-miner/sectors.go:320]: Why?

**Review by arajasek — APPROVED** [review]
LGTM with one q

File: cmd/lotus-miner/sectors.go (hunk 1 above)
Comment (magik6k) [reply cmd/lotus-miner/sectors.go:320]: This was getting bottlenecked by a single-threaded json encoder in websocket mode. In http mode all those requests are running on entirely separate goroutines, and don't get blocked on json encoding/decoding

Comment (arajasek) [reply cmd/lotus-miner/sectors.go:320]: Interesting, thanks. Say that in a comment, please, then merge?

**Pull request merged**